### PR TITLE
Snowmap Salvage Project PR#6 - Deer

### PR DESCRIPTION
**Pull request by MadmanMartian**
Salvages deer from #14794

Deer wander around, but get spooked by just about anything unless you get yourself geared up appropriately.

Also adds 

- the name_mob proc, as I saw it was copypasted from wolves.dm
- MOB_NO_LAZ mob property flag, so it definitely can't be revived via lazarus injection

### CODE REVIEW

**Comment by ghost** [comment]
heck yes
image

**Comment by Probe1** [comment]
My bad on the conflicts in all honesty I thought yours was more recent on the list and I didn't really check.

**Comment by MadmanMartian** [comment]
it's a binary conflict, shit happens

File: code/modules/mob/living/simple_animal/hostile/deer.dm [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+		Otherwise, if you're wearing clothing made of deer, or are holding an apple, they are less likely to get spooked
+	*/
+		var/mob/living/carbon/human/H = target
+		var/spook_prob = 30
+
+		if(H.m_intent == "run") //Don't run while stalking deer
+			spook_prob = 70
+		if(istype(H.get_item_by_slot(slot_wear_suit), /obj/item/clothing/suit/leather/deer))
+			spook_prob -= 10
+		if(istype(H.get_item_by_slot(slot_head), /obj/item/clothing/head/leather/deer))
```
Comment (Probe1) [inline code/modules/mob/living/simple_animal/hostile/deer.dm:45]: Let me in I'm a fairy.  You believe in fairys right?

**Comment by Probe1** [comment]
Not gonna lie the sprite for the deer mask looks like a mess 
image

Too much detail in too small an area.

**Comment by Probe1** [comment]
attackby() with an apple on help intent still results in the string ```This weapon is ineffective, it does no damage.```

**Review by Probe1 — COMMENTED** [review]
Find a way to surpress the ``` This weapon is ineffective, it does no damage. ``` string when feeding apples to deer and this is good to go in my eyes.

**Comment by MadmanMartian** [comment]
That deer mask isn't associated with this PR, it's from a reindeer version of the horsemask spell.

**Comment by Probe1** [comment]
HIT IT

**Pull request merged**